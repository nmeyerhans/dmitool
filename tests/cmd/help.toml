bin.name="dmitool"
args="--help"
