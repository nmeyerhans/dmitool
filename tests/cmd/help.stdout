Decodes and prints system information from the SMBIOS

Usage: dmitool [OPTIONS]

Options:
  -0                   print table 0 via the /sys/firmware/dmi/entries interface
  -t, --table <TABLE>  print the given table via the /sys/firmware/dmi/tables
  -e, --entrypoint     read SMBIOS entrypoint
  -d, --debug          enable debug output
  -h, --help           Print help
  -V, --version        Print version
